packages/ui-components/src/components/LoginDialog/LoginDialog.tsx

```diff
@@ -21,14 +21,23 @@ const LoginDialog: React.FC<Props> = ({ onClose, open = false }) => {
   const makeLogin = useCallback(
     async (username?: string, password?: string): Promise<LoginBody | void> => {
       // checks isEmpty
-      if (isEmpty(username) || isEmpty(password)) {
+      if (!username || !password || isEmpty(username) || isEmpty(password)) {
         dispatch.login.addError({
           type: 'error',
           description: i18next.t('form-validation.username-or-password-cant-be-empty'),
         });
         return;
       }
 
+      // checks min username and password length
+      if (username.length < 2 || password.length < 2) {
```

Comment:
This might be a future parameter, maybe others wants increase or set different rules.

Reply:
Parameters for user/pwd rules make sense for `adduser`, usually. You want to keep login as open as possible to avoid giving any edge to any malicious actor.